phone-test/src/pages/CallDetail/CallDetail.tsx

```diff
@@ -31,7 +73,7 @@ export const CallDetail: React.FC<CallDetailProps> = ({ call, onClick }) => {
       <Box
         minWidth="1"
         key={call.id}
-        bg="black-a30"
+        bg={call.is_archived ? '#C9DFDB' : 'black-a10'}
```

Comment:
Changing the background color of the call card to facilitate visual differentiation of a call status: archived or non archived